guava/src/com/google/common/primitives/Booleans.java

```diff
@@ -122,7 +122,7 @@ public static int hashCode(boolean value) {
    * @param a the first {@code boolean} to compare
    * @param b the second {@code boolean} to compare
    * @return a positive number if only {@code a} is {@code true}, a negative number if only {@code
-   *     b} is true, or zero if {@code a == b}
+   *     b} is {@code false}, or zero if {@code a == b}
```

Comment:
I appreciate the edit to add code formatting, but this should be `{@code true}`

Reply:
okay will change that. Thankyou